pom.xml

```
@@ -121,6 +122,12 @@
   </properties>
 
   <dependencies>
+      <dependency>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-client-api</artifactId>
+          <version>${hadoop.client.version}</version>
+          <scope>provided</scope>
+      </dependency>
```

issue (blocking): if  you want to add this dependency, use the version consistent with the other two hadoop  dependencies defined here already

```suggestion
      <dependency>
          <groupId>org.apache.hadoop</groupId>
          <artifactId>hadoop-client-api</artifactId>
          <version>${hadoop.version}</version>
          <scope>provided</scope>
      </dependency>
```

Another alternative:
1. Delete hadoop-client-api introduced by this PR, and hadoop-common on L130
2. Change the scope for hadoop-mapreduce-client-core to provided